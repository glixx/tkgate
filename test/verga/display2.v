module top;

  reg [127:0] a;

  initial
    begin
	$display(1652763623);
	$display("%d",1652763623);
	$display(1652762343623232);
	$display("%d",1652762343623232);
	$display(1652762343623231827368712362);
	$display(98723498232342349823741652762343623231827368712362);
	a = 1652763623;
	$display(a);
    end

endmodule
