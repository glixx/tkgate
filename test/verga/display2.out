32'd1652763623
1652763623
64'd1652762343623232
1652762343623232
96'd1652762343623231827368712362
192'd98723498232342349823741652762343623231827368712362
128'h62832be7
